app/templates/views/platform-admin/reports.html

```diff
@@ -1,7 +1,7 @@
 {% extends "views/platform-admin/_base_template.html" %}
 
 {% block per_page_title %}
-  Platform admin
+  Admin panel
```

Comment:
```suggestion
  {{ _('Admin panel') }}
```
To translate the page title